### PR TITLE
Home Assistant Red Alert app

### PR DESCRIPTION
**Pull request by danielrosehill**
I tried but failed to figure out what was going on with the markdown in VSCode (see the TOS where it's added a bunch of indents that I can't seem to get rid of. Sorry .. but I also wanted to use this PR to see if adding Home Assistant components was okay as there are a few that I'd be happy to send in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,36 +17,37 @@ A curated list of awesome Open Source Software made in Israel, and by Israelis.
...
-- [Companies](#companies)
-- [Organizations](#organizations)
-- [Podcasts](#podcasts)
-- [Individuals](#individuals)
-- [Events](#events)
-- [Meetups / User Groups](#meetups--user-groups)
-- [Israeli Tech AwesomeLists](#israeli-tech-awesomelists)
-- [Contributing](#contributing)
-- [License](#license)
+- [Table of Contents](#table-of-contents)
```
Comment (lirantal) [inline README.md:20]: @danielrosehill can you please remove this change to the table of contents? that's not needed

File: README.md [hunk 2]
```diff
@@ -217,6 +218,7 @@ ic research to an artifact that engineers can deploy easily. ![GitHub last commi
 * [PySnooper](https://github.com/cool-RR/PySnooper) - a Python debugging aid. "Never use print for debugging again." ![GitHub last commit](https://img.shields.io/github/last-commit/cool-RR/PySnooper?style=flat-square) ![GitHub top language](https://img.shields.io/github/languages/top/cool-RR/PySnooper?style=flat-square)
 ![GitHub stars](https://img.shields.io/github/stars/cool-RR/PySnooper?style=flat-square)
 * [Redash](https://github.com/getredash/redash) - Data visualization tool for building dashboards that connect to multiple data sources. ![GitHub last commit](https://img.shields.io/github/last-commit/getredash/redash?style=flat-square) ![GitHub top language](https://img.shields.io/github/languages/top/getredash/redash?style=flat-square)
+* * [Red Alert (Home Assistant Integration)](https://github.com/idodov/RedAlert) - Home Assistant component for providing alerts from the Israeli Home Front Command system and making them available as sensors in Home Assistant. ![GitHub last commit](https://img.shields.io/github/last-commit/idodov/RedAlert?style=flat-square) ![GitHub top language](https://img.shields.io/github/languages/top/idodov/RedAlert?style=flat-square)
```
Comment (lirantal) [inline README.md:221]: Looks great and I'd love to add it.

Comment (lirantal) [inline README.md:221]:
```suggestion
* [Red Alert (Home Assistant Integration)](https://github.com/idodov/RedAlert) - Home Assistant component for providing alerts from the Israeli Home Front Command system and making them available as sensors in Home Assistant. ![GitHub last commit](https://img.shields.io/github/last-commit/idodov/RedAlert?style=flat-square) ![GitHub top language](https://img.shields.io/github/languages/top/idodov/RedAlert?style=flat-square)
```

**Review by lirantal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danielrosehill) [reply README.md:20]: @lirantal - It's automatic indentation from VS Code indenting everything by one space forward whenever I save the file. Driving me crazy trying to stop it from happening. Let me see if I can figure it out and create a new PR (you might be able to see from the commits that I already tried once unsuccessfully!)

Comment (lirantal) [reply README.md:20]: the problem is that it's indenting one level deep.

feel free to close this PR and make changes only to the other section in the README to add your project and submit a new PR with it.

Comment (danielrosehill) [reply README.md:20]: Will do!

File: README.md (hunk 2 above)
Comment (lirantal) [reply README.md]: @danielrosehill note that we didn't end up merging this change

**Pull request closed without merging**